### PR TITLE
added cancer type to civic query

### PR DESCRIPTION
**Pull request by HomoPolyethylen**
**PR Checklist**

- [x] This comment contains a description of changes (with reason)
- [x] Referenced issue is linked
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.rst` is updated

**Description of changes**

* added CL option to specify cancer type when querying CIViC (`-c` `--cancer`). #27 

**Technical details**

  * The cancer type is used to filter the evidence items returned from civicpy.
  * evidence is included if:
    A) the evidences cancer type (in civic: disease) matches the specified cancer type
    B) the evidences cancer type (disease) is a supergroup of the specified cancer type

**Additional context**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -180,6 +180,7 @@ A typical command to query the `Clinical Interpretations of Variants in Cancer -
         -v input_file.vcf \
         -o outdir \
         -g ref_genome [GRCh37, GRCh38, NCBI36]
+        -d 'breast adenocarcinoma'
```
Comment (mapo9) [inline docs/usage.rst:183]: should be c now?

```suggestion
        -c 'breast adenocarcinoma'
```

**Review by mapo9 — CHANGES_REQUESTED** [review]

**Review by famosab — APPROVED** [review]
LGTM after changing Marks comment.

File: docs/usage.rst (hunk 1 above)
Comment (HomoPolyethylen) [reply docs/usage.rst:183]: praise the review process! the `commit suggestion` button does not work ("Could not apply suggestion"). Could this be due to branch protection of dev? Do I really need to make a new PR to dev, just to change that d into a c? :skull:

Comment (mapo9) [reply docs/usage.rst:183]: No:

1. install gh 
2. Go to your clone
3. do gh pr checkout 29
4. do the changes
5. push

you will automatically push to this pr

Comment (HomoPolyethylen) [reply docs/usage.rst:183]:
```
(nf24) [zxmgc83@thanos querynator]$ gh pr checkout 29
branch 'dev' set up to track 'origin/dev'.
Switched to a new branch 'dev'
(nf24) [zxmgc83@thanos querynator]$ git pull
Already up to date.
```
made changes
```
(nf24) [zxmgc83@thanos querynator]$ git status
On branch dev
Your branch is ahead of 'origin/dev' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
(nf24) [zxmgc83@thanos querynator]$ git push
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 64 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 356 bytes | 356.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/dev.
remote: error: Changes must be made through a pull request.
To github.com:qbic-pipelines/querynator.git
 ! [remote rejected] dev -> dev (protected branch hook declined)
error: failed to push some refs to 'github.com:qbic-pipelines/querynator.git'
```

**Review by mapo9 — APPROVED** [review]

**Pull request merged**